pages/posts/roadmap-2024.md

```diff
@@ -0,0 +1,219 @@
+
+Our sustained focus is on shipping the minimum network functionality necessary to serve all of the Execution layer data.  This includes the three independent Portal clients which implementation the three sub-protocols that house and serve this data, tooling such as our network health monitor (GladOS), interoperability testing via Hive, and the "bridge" node functionality which is responsible for pushing data into the Portal Network.
+
+Additionally, we allocate a significant portion of our time to support the practical adoption of this technology by client and app teams around the world.
+
+## Portal mission & vision
+
+The overall vision for Portal Network is to ***provide the infrastructure needed for anyone to access Ethereum data in a decentralized way with minimal hardware and bandwidth requirements.***
+
+Portal Network is a public good that aims to become a core piece of infrastructure for many Ethereum-based applications. It will provide decentralized API access to Ethereum execution and consensus data and support several pathways to lightweight Ethereum clients. 
+
+There are currently three primary sub-protocols that the Portal team are committed to shipping as priority deliverables in 2024:
+
+- History network
+- State network
+- Beacon network
+
+These sub-protocols are prioritized because together they provide verifiable access to the full Ethereum dataset in a manner that is fully decentralized and accessible on resource constrained devices. 
+
+Specifically: 
+
+- The History sub-protocol enables Ethereum execution clients to expire historical data via EIP-4444, safe in the knowledge that the data is available on request from the Portal Network. 
+- The State sub-protocol gives light clients access to Ethereum stata data, exposing methods such as `eth_getProof` that eliminate the need for trusted third party RPC providers. 
+- The Beacon sub-protocol gives light clients access to the latest block headers so that they can be confident they are following the canonical Ethereum chain.
+
+Shipping these features will enable ***major upgrades for the Ethereum ecosystem as a whole***. These featues have been in development for several years and have been through several rounds of refinement and course correction. Now Portal Network is on the brink of shipping these sub-networks into production. 
+
+This is why ***2024 is a set to be a standout year for Portal Network.***
+
+Longer term, there are proposals for more sub-protocols that serve granular transaction data, blob data, proofs etc, but these are out of scope for 2024.
+
+
+## High level goals
+
+### 2024 
```

Comment:
I understand that these goals are goals and not commitments.

But are the key results to be interpreted from good to great based on their numbering?

Reply:
The order is arbitrary - there is no prioritisation to be inferred from the numbering.